### ISSUE TITLE
Can't enable all checkers that may be run for shell scripts

### ISSUE THREAD

**Issue by EmilyGraceSeville7cf** [flycheck/flycheck#1967]
This template is for **user support**.  If you are reporting an actual Flycheck bug, please use the relevant issue template instead.

## Checklist
- [ ] I have searched existing issues and StackOverflow for solutions.
- [x] I have read the [Flycheck manual][].
- [x] I have read the [Troubleshooting Guide][].
- [ ] I have checked my issue has not been reported yet.

## Problem description

I wanna enable `sh-posix-dash` and `sh-posix-dash` checkers:

![image](https://user-images.githubusercontent.com/42812113/182295326-07415c40-cfc6-4068-92f8-e876aeec61f0.png)

but when I press `C-u C-c ! x` I obtain `x is undefined` or `X is undefined`. I don't understand why it happenes.

## Steps to reproduce

- Press (`shift`+`1`+`x`) or (`shift`+`1` and then `x`) in `*Flycheck checkers*` buffer to obtain `x is undefined` or `X is undefined`.

and nothing helped.

- [ ] I have read https://emacs.stackexchange.com/questions/28429/how-do-i-troubleshoot-emacs-problems

## System configuration
```
Paste the output of `M-x flycheck-verify-setup` here.
```

Emacs configuration:
- [x] Plain Emacs / Custom configuration
- [ ] Spacemacs
- [ ] Doom Emacs
- [ ] Other shared configuration

## Additional notes

```
Flycheck version: 33snapshot (package: 20220729.1039)                                                                                          
Emacs version:    28.1                                                                                                                         
System:           x86_64-pc-linux-gnu                                                                                                          
Window system:    nil
```

[Flycheck manual]:  https://www.flycheck.org/en/latest/user/troubleshooting.html
[Troubleshooting guide]:  https://www.flycheck.org/en/latest/user/troubleshooting.html

**Comment by EmilySeville7cfg** [comment]
@lunaryorn, any thoughts? (Sorry for tagging you but I need answer from anybody who knows it.)
